.env

```diff
@@ -21,4 +21,7 @@ OKP4_ARBITRAGE_URL='https://www.arbitrage.org'
 OKP4_ARBITRAGE_RULES_URL='https://www.arbitrage.org/arbitrage/#REGLEMENT_ARBITRAGE'
 OKP4_COOKIE_POLICY_URL='https://okp4.network/cookie-policy'
 GRAPHQL_URI='https://api.nemeton.staging.okp4.network/graphql'
-OKP4_TASKS_SIDH_5_TYPEFORM_URL='https://okp4.typeform.com/Sidh-task5?utm_source=xxxxx'
\ No newline at end of file
+OKP4_TASKS_SIDH_5_TYPEFORM_URL='https://okp4.typeform.com/Sidh-task5?utm_source=xxxxx'
```

Comment:
Don't use `utm_source` query parameter with this value, it makes no sense imho. Or you can position a relevant value that reflects the source of the traffic.